src/main/java/com/softserve/messaging/service/MailTemplateDirector.java

```diff
@@ -0,0 +1,31 @@
+package com.softserve.messaging.service;
+
+import com.softserve.messaging.dto.MailTemplateDto;
+import com.softserve.messaging.model.MessageHandler;
+import com.softserve.messaging.util.UtilMailMessage;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
```

Comment:
you marked the field as `final`. Isn't it reasonable to use `@RequiredArgsConstructor` then?

Reply:
my bad, misstyped